deploy/test/deploy_test.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:deploy/deploy.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('calculate', () {
```

Comment:
should we delete this? I think it's auto generated from when the project was created.